src/marionette/transport.cr

```diff
@@ -63,7 +66,7 @@ module Marionette
           end
         end
       end
-
+      Log.trace { "(#{@id}) Marionette receive #{raw}" }
```

Comment:
It i can be really big, can we limit that by some appropriate size?

Reply:
If it is really important to debugging just close conversation as resolved.

Reply:
I specially set that this log level trace. Our puppet logs are not included. I have not noticed large logs at home. I think it's worth leaving. if it interferes with local launches then we will delete) @dammer , what do you say?

Reply:
agree with Damir, not sure why we need all this in log. I think log about 100-200 bytes would be enough.

Reply:
if you insist) Fixed